app/views/dashboard/districts/index.html.erb

```diff
@@ -1,14 +1,19 @@
-
-<div class="mb-2 pt-8 px-4 md:flex md:items-center md:justify-between md:max-w-6xl md:m-auto md:pt-24">
-  <div>
-    <div class="flex items-center mb-4">
-      <div class="mr-2">
-      <% @districts.each do |district| %>
-        <p class="font-roboto font-bold text-md text-blue-200">
-          <%= link_to district.name, dashboard_district_path(district.slug) %>
-        </p>
+<div class="container">
+  <div class="row">
+    <div class="col-sm-5 col-lg-7">
+      <% @organizations.each do |organization| %>
+      <section>
```

Comment:
@rsanheim `section` is missing top and bottom margins for spacing between cards, `margin: 16px 0;`. Let me know if you'd rather I add it myself.

Reply:
Sure, you can add it to this PR 👍

Reply:
@rsanheim pulled this branch and margin is already included in `card` class